charters/DataProcessingService/DataProcessingPipelines.md

```diff
@@ -0,0 +1,70 @@
+* Establishing and operating automated practices around scientific testing of pipelines
+* Performing outreach to users on the usability of data in downstream applications 
+* Performing outreach to developers on pipeline development in the HCA DCP 
+* Suggesting, but not selection of, new standardized pipelines for the HCA DCP 
+* Working with the community to provide feedback on metadata standards for raw data in the HCA DCP to provide the required information to enable processing of the raw data.
+* Working with the metadata team and data consumers to ensure the metadata provided about pipelines and the analysis results meet the needs of downstream users
+
+### Scientific "guardrails"
+
+* Initial versions of standardized pipelines are reviewed by the AWG.  
+* Changes to standardized pipelines that affect the scientific output are reviewed by the AWG, engineering updates that do not affect the scientific output are not reviewed by the AWG.  
+* Selection of standardized pipelines to work on are approved by the HCA DCP Governance Group (this does not apply to community pipelines, community pipelines are on-boarded as a service to the scientific community). Prioritization of the work to develop selected pipelines is performed by the Data Processing Service team.  
+
+## Out of Scope
+* Educational training or user assistance on how to perform data analysis with outputs of pipelines  
+* Recommendations or demonstrations of downstream tool use beyond initial data loading  
+* Novel methodological development is not performed by the data processing team unless it is essential to the pipeline, it does not existent in community tools, and it represents minimal effort to implement  
+* Software engineering 3rd party tools
+
+## Milestones and Deliverables
+__2018:__ Two data processing pipelines in production (Smart-seq2 and 10xv2 human data).  
+__2019:__ Adding at least three more data processing pipelines, including 10xv2 snRNA-seq and mouse pipelines for all current analysis pipelines. Aim to have at least one _in situ_ transcriptomics pipeline in this year. Aim for one of these pipelines to be a community pipeline. Improved automated scientific testing and benchmarking of analysis pipelines.
+__2020:__ Continued addition of analysis pipelines.  
+
+## Roles
+### Project Lead
+[Timothy Tickle](mailto:ttickle@broadinstitute.org)
+
+### Product Owner
+[Kylee Degatano](mailto:kdegatano@broadinstitute.org)
+
+### Technical Lead
+[David Shiga](mailto:dshiga@broadinstitute.org)
+
+## Communication
+[Team email](mailto:pipelines-team@data.humancellatlas.org): pipelines-team@data.humancellatlas.org
+__HumanCellAtlas#mintteam:__ team discussions, a place to directly communicate to the team, who works on the service and the pipelines, as needed  
+__HumanCellAtlas#secondary-analysis:__ community-focused space for conversations around data processing pipelines that run in the DCP 
+
```

Comment:
Formatting nit: you will need an extra line after each bullet item to keep them from flowing into each other (on many markdown interpreters).

Reply:
Ah thanks for the tip! Will update.

Reply:
You can also use an editor like [vscode](https://code.visualstudio.com/) to preview markdown which I find helpful.